rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -54,26 +117,133 @@ paths:
           $ref: '#/components/responses/NotImplemented'
         default:
           $ref: '#/components/responses/DefaultError'
-
+  /transform/{TransformId}:
+    parameters:
+      - $ref: '#/components/parameters/TransformId'
+    get:
+      summary: Poll transformation request status
+      description: >
+        After posting a transformation request, the client can use the returned URI to poll for the transform's completion at this endpoint.
+      operationId: poll
+      tags:
+        - transform
+      responses:
+        '200':
+          description: The transform is still in progress.
+          headers:
+            Content-Version:
+              $ref: '#/components/headers/ContentVersion'
+        '201':
+          description: Transform complete. Returning metadata locations.
+          headers:
+            Content-Version:
+              $ref: '#/components/headers/ContentVersion'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    format:
+                      $ref: '#/components/schemas/MetadataFormat'
+                    location:
+                      description: The URI for where the metadata is located.
```

Comment:
Are we okay with the response body containing a list of objects with the MetadataFormat (i.e. irm or cst) as one property and the URI as another? I felt a map would be simpler, but I couldn't think of a way to do a map and store the "supported metadata formats" as a single enumerated list in the API. i.e. the keys of the map could not be values from an `enum`.

Reply:
I actually like your idea of a structure holding all of the above. It doesn't prevent duplication as a map would do but it does allow for a reusable model for metadata information. We have the same concept in rep/sync right now and I suspect CDR will have a similar concept at some point since the system level info we need to keep track for a metadata is similar across the various products.